targets/chrome-common

```diff
@@ -31,26 +31,52 @@ REQUIRES='x11'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-# Only download if it's not already installed, since it auto-updates
 pkg="google-chrome-$CHANNEL"
-if [ -n "`list_uninstalled '' "$pkg"`" ]; then
-    # Chrome needs hicolor-icon-theme to configure/install
-    install hicolor-icon-theme
-
-    url="https://dl.google.com/linux/direct/${pkg}_current_$ARCH.$PKGEXT"
-    tmp="`mktemp crouton.XXXXXX --tmpdir=/tmp`"
-    addtrap "rm -f '$tmp'"
-
-    wget "$url" -O "$tmp"
-
-    # Remove other providers of google-chrome, then install the package
-    for c in stable beta dev; do
-        c="google-chrome-$c"
-        if [ -z "`list_uninstalled '' "$c"`" ]; then
-            remove "$c"
-        fi
-    done
-    install_pkg "$tmp"
+
+# Chrome needs hicolor-icon-theme to configure/install
+install hicolor-icon-theme
+
+# Use the package manager for gentoo, it maintains chrome as well
+if [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    # Rename google-chrome-stable to google chrome to match the ebuild
+    if [ "$CHANNEL" = 'stable' ]; then
+        GENTOO_PKG="www-client/google-chrome"
+    else
+        GENTOO_PKG="www-client/$pkg"
+    fi
+
+    # Keyword Chrome
+    keyword_package "$GENTOO_PKG" "$pkg"
+
+    # Accept the license
+    accept_license "$GENTOO_PKG" google-chrome "$pkg"
```

Comment:
> I suppose I could ask the user if they accept the license as part of the install sequence? Is it okay to have this interaction in the install process?

That's one possibility yes. We do have some automated tests though, so you'd need to make sure these interactions can be automated.

> To be fair though, shouldn't that really happen on the other distros too? I have to admit, I haven't tried installing chrome using one of the existing distros, maybe this is part of the install process?

That's a good point, we don't do anything on other distros. I guess it's fine this way then (especially since you take care of listing the licenses at the end of the install process).